### DISCUSSION TITLE
[按钮功能] Notion支持设置/button 希望在网页中支持

### DISCUSSION THREAD

**Discussion by aistart** [tangly1024/NotionNext#2107] [Ideas]
**描述bug**
采用/button在Notion page中添加一个按钮，page中能正确实现：
比如就选择设置Show confirmation后，在notion中点击该按钮后就有弹窗。
但是在NotionNext部署之后，就是一片空白。没有看到button。

**复现步骤**
同上，很容易复现

**截图**
Notion上的设置/button
![屏幕截图 2023-10-11 080141](https://github.com/tangly1024/NotionNext/assets/55378001/7d09f976-9a49-4d7b-90dc-eb4d3854d296)
Notion上点击button的结果
![屏幕截图 2023-10-11 080219](https://github.com/tangly1024/NotionNext/assets/55378001/78780470-ff54-4912-b515-ef288bd670e3)

**期望的正常结果**
希望按这个步骤，正常操作结果是：能在NotionNext的web页面上看到同样效果。
注意到Notion里面/button 标记了new，是不是因为这是新功能，在NotionNext上还没有实现？

**环境**
- 操作系统: [例如.windows]
- 浏览器 [例如. chrome, edge]
- NotionNext版本 [4.0.15]
- 主题 [例如. matery]

**Comment by tangly1024** [comment]
新功能，NotionNext 暂不支持
